apps/frontend/src/components/modals/AstrologyModal.tsx

```diff
@@ -402,6 +402,7 @@ const AstrologyModal = ({
     if (step === steps?.length - 1) {
       onClose?.();
       setIsOpen(false);
+      setStep(0);
```

Comment:
Note: after completing, the step will be reset to 0 so that you'll edit from the start. This seemed more intuitive than editing in reverse order.